src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -140,6 +140,15 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
         assertCommandSuccess(command, actualModel, expectedCommandResult, expectedModel);
     }
 
+    /**
+     * Compares to a command that takes in an internship application to display
+     */
+    public static void assertCommandSuccess(Command command, Model actualModel, String expectedMessage,
+                                            Model expectedModel, InternshipApplication internshipApplication) {
```

Comment:
The styling indentation here is a bit off

Reply:
Okay will fix this thanks for the catch

Reply:
Please use the auto formatter provided by intellij